db/index.js

```diff
@@ -12,3 +12,19 @@ const client = new Client({
 client.connect()
   .then(() => console.log('Connected to Postgres server!'))
   .catch((error) => console.log('Could not connect to Postgres server:', error));
+
+const testFunction = function () {
+  return new Promise((resolve, reject) => {
```

Comment:
@jabrake I don't understand the use of the Promise constructor. client.query should return a promise.

Reply:
Correct - my thinking here: because I'm invoking `testFunction` in server/index.js (and console.logging what it returns), if I don't wrap client.query in a Promise, it logs a <Pending> Promise since that gets executed before the query to the DB is complete.

But am I overthinking it? Or is there a simpler way to do it?

Reply:
@jabrake Please do correct me if I'm wrong but my understanding is that you need to wrap everything with another Promise because that's how data and error are passed up the Promise chain? I don't see how you could return those variables unless you use Resolve and Reject functions provided by the Promise constructor. Or am I missing something here?

Reply:
@jabrake @tchitrakorn : If you just return client.connect inside the testFunction, it should return a promise when testFunction is invoked. We can discuss it today, if it doesn't make sense.

Reply:
Oh that does make sense actually. Yeah I just need my function to return a Promise so that should work! I'm going to merge this and then update that next (after I complete the ETL process)

Reply:
@farhanali3193 It took me a while to realize that this is possible, ahah. Thank you!